### DISCUSSION TITLE
How do i insert the websocket and and localhost into the assembly?

### DISCUSSION THREAD

**Discussion by c4palot** [RealMCoded/Rec.js#9] [Q&A]
when i try to do it i get this "/localhost:2010//api/versioncheck/v3?v=20171002_TRAILER" 404 - 2.196 ms

**Comment by RealMCoded** [comment]
Hi there! I assume this might have been a typo in the server IP. Please check to see if your assembly's RecNet IP definitions look something like the screenshot attached (the parts in the red circle).

![image](https://github.com/RealMCoded/Rec.js/assets/34777184/7c4e0647-65c2-4712-b4b0-980627f09a36)

Also, some other helpful info would be nice. For example:
- Rec.js Server Version (e.g: 0.1.5-pre2 (commit ABCD123))
- Have you done any modifications to the server software at all?
